pennylane/shadows/classical_shadow.py

```diff
@@ -282,7 +282,7 @@ def pauli_list_to_word(obs):
             return self._convert_to_pauli_words_with_pauli_rep(pr, num_wires)
 
         raise ValueError(
-            "Observable must have a valid pauli representation. Recevied {observable} with observable.pauli_rep = {pr}"
+            f"Observable must have a valid pauli representation. Received {observable} with observable.pauli_rep = {pr}"
```

Comment:
What do we expect `observable.pauli_rep` to be at this point other than `None`?

Reply:
None that I am aware of. You want me to remove the last bit?

Reply:
Personally I think it's fine, it will just say `None` in the error anyway. Optionally, you can change it explicitly to be the following, but it has no impact.
```suggestion
            f"Observable must have a valid pauli representation. Received {observable} with observable.pauli_rep = None"
```